[Rewrite] Function Wikis - #2 (#203)
NilPointer-Software/bdfd-wiki: src/bdscript/userJoined.md

# $userJoined
Returns the server joining date of a given user.

## Syntax
```
$userJoined[User ID;(Format)]
```

### Parameters

> - `User ID` : The ID of the user whose join date is to return.

Reviewer: ```suggestion
- `User ID` : The user whose join date will be returned.
```
